frontend/src/components/details/navbar.css

```diff
@@ -1,30 +1,90 @@
 /* Style the navigation menu */
 .navbar {
     width: 100%;
-    background-color: #468c46;
-    overflow: auto;
+    border-radius: 25px;
+    height: 80px;
+    padding-top:5px;
+    color:  #F0F0F0;
+    /* border: 3px solid var(--global--button--color); */
+    /* background-color: var(--global--panel--color); */
+}
+
+/* -webkit-radial-gradient(  bottom right, #4D217A,#1B1534 ); */
+:root{
+    --global--button--color:#836387;
+    --global--button--text--color:#F0F0F0;
+    --global--panel-title--FontSize: 22px;
+    --global--text-subtitle--FontSize: 19px;
 }
```

Comment:
Consider maybe moving the variables to the variables file and adding a comment on what these variables are for?

Reply:
I did not do the documentation I will do it today. 😄